### ISSUE TITLE
.asc instead of .pubkey

### ISSUE THREAD

**Issue by maks-rafalko** [dockerized-php/infection#3]
Dockerfile downloads the old file which does not exist anymore:

https://github.com/dockerized-php/infection/blob/d87e6e6ffce4bba749128016bcad6d819070e294/latest/Dockerfile#L23

Please see  https://infection.github.io/guide/installation.html

```
wget https://github.com/infection/infection/releases/download/0.11.5/infection.phar
wget https://github.com/infection/infection/releases/download/0.11.5/infection.phar.asc

chmod +x infection.phar
```

**Comment by tommy-muehle** [comment]
Hi @borNfreee,

thanks for the hint! I totally missed that. But it's now fixed. I also added the GPG verification with the last commit.

Best,
Tommy

**Comment by tommy-muehle** [comment]
Fixed with 77b18f64d09d466c023efb4e89fa2c91fde7f029

**Comment by maks-rafalko** [comment]
👍
